api/actions/v1/actions.proto

```diff
@@ -434,7 +433,7 @@ service ActionsService {
     };
   }
   // CancelAction stops an Action.
-  rpc CancelAction(CancelActionRequest) returns (CancelActionResponse) {
+  rpc CancelAction(CancelActionRequest) returns (google.protobuf.Empty) {
```

Comment:
Bad thing about `Empty` type is that it cuts off extensibility. In the future there will be no option to return, for example, action progress that was done, or something else.

Reply:
Agree with Artem, let's drop usage of `Empty` type

Reply:
Yes: https://github.com/percona/pmm/pull/731/files

Reply:
> another POV - have we really had such need in the past of such extensibility?

Yes, we have changed the request and response types in the past. This was OK, since extending protobufs doesn't break anything.

@artemgavrilov has mentioned that we'd rather be consistent in using empty messages everywhere, otherwise we won't be able to establish a guide clearly stating when to use what.